docs/logs/redaction/MyRedactionProcessor.cs

```diff
@@ -23,15 +23,9 @@ internal class MyRedactionProcessor : BaseProcessor<LogRecord>
 {
     public override void OnEnd(LogRecord logRecord)
     {
-        if (logRecord.State == null)
+        if (logRecord.Attributes != null)
```

Comment:
adding a comment to the PR for anyone wondering: This change is not required really (except to silence warning about using obsolete. The change is backcompatible, so if a Redactor is looking at State or Statealues as before, it'll continue to work, and there is no risk of "Oh damn, I missed to redact secret attributes".